include/picongpu/param/pusher.param

```diff
@@ -62,6 +62,15 @@ namespace picongpu
         namespace sqrt_Vay = precision64Bit;
     }
 
+    namespace particlePusherHC
```

Comment:
optional: in other places, e.g. the current deposition methods, there is no abbreviation used.

Reply:
I would also prefer the full name everywhere instead of HC. (internally in the implementation you could make an alias if the names are too wordy).